docs/developer_documentation/app-development.md

```diff
@@ -0,0 +1,130 @@
+# MOLGENIS App store
+
+You can make your own app in MOLGENIS using the appstore.
+
+You can perform the following steps to get your app in the App store.
+
+- [Create a project](#create-a-project)
+- [Update your build configuration](#update-build-configuration)
+- [Update your language specific configuration](#update-your-language-specific-configuration)
+- [Create a freemarker-template](#create-a-freemarker-template)
+- [Upload your app](#upload-your-app)
+- [Active the app](#active-the-app)
+
+## Create a project
+You can create a javascript project in a language you like. 
+We develop our own apps in VUE so most documentation will point out the VUE-configuration.
+
+## Update your build configuration
+You must update build configuration to build a zip-file which you can upload in MOLGENIS. 
+At the moment we only describe how to do this in webpack.
```

Comment:
Remove this line. If they use something else than webpack, they can look up their build tool specific alternatives